### PR TITLE
[16.0][MIG] pos product display default code: migration to 16.0

### PR DESCRIPTION
**Pull request by suker**
PR for migration from v15.0 to v16.0

related task: #849

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -101,7 +101,7 @@ repos:
       - id: pyupgrade
         args: ["--keep-percent-format"]
   - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
+    rev: 5.12.0
```
Comment (ivantodorovich) [inline .pre-commit-config.yaml]: Pls rebase, this has been changed directly on `16.0` now ;)

Comment (suker) [reply .pre-commit-config.yaml]: okey! thanks :D

**Review by chienandalu — APPROVED** [review]
LGTM :+1: Thanks for the improvements :)

**Pull request merged**